src/app/app.module.ts

```diff
@@ -14,6 +14,7 @@ import { SharedModule } from './shared/shared.module';
 import { CampaignModule } from './campaign/campaign.module';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { CampaignListModule } from './campaign-list/campaign-list.module';
+import { ReactiveFormsModule } from '@angular/forms';
```

Comment:
I am not sure the ReactiveFormsModule should be imported at the app.module level. Why not just in the daterange.module?